### PR TITLE
feat: implement BatteryCC v3

### LINKED ISSUE #1156: Implement Battery CC V3
Battery CC V3 adds a single field to the Battery Report command:
https://github.com/zwave-js/node-zwave-js/blob/master/specs/SDS13782%20Z-Wave%20Management%20Command%20Class%20Specification.pdf

**TODOs:**
- [x] Parse the new field
- [x] Include it in logging
- [x] update `@implementedVersion` decorator

### PR DESCRIPTION
**Pull request by varet80**
Add support for Battery CC v3 (upgrade from V2

Fixes #1156 

@AlCalzone I will need your review, If I missed something, relatively tiny change!

### CODE REVIEW

File: packages/zwave-js/src/lib/commandclass/BatteryCC.ts [hunk 1]
```diff
@@ -160,6 +161,7 @@ needs to be replaced or charged: ${
 							batteryStatus.rechargeOrReplace!
 						]
 					}
+is low temperature               ${batteryStatus.lowTemperatureStatus}
```
Comment (AlCalzone) [inline packages/zwave-js/src/lib/commandclass/BatteryCC.ts:164]: I would call this `temperature is low`

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Just a small change.

And you need to format the file. Either install the recommended VSCode extensions (enabling format-on-save) or run
```
yarn run lint:ts --fix
```

**Comment by varet80** [comment]
there are linting errors coming from other files! which confused. i did run lintfix but still some where not recovereable! and some more changes arrived with linting

Fixed also the change requested

File: packages/zwave-js/src/lib/commandclass/BatteryCC.ts (hunk 1 above)
Comment (varet80) [reply packages/zwave-js/src/lib/commandclass/BatteryCC.ts:164]: done! added also `:`

**Comment by AlCalzone** [comment]
I can check later. That usually happens when you have different dependency versions than CI.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**